src/app/components/LinkedData/index.tsx

```diff
@@ -111,12 +111,32 @@ const LinkedData = ({
 
   const hasByline = !!bylineLinkedData;
 
-  const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
-    bylineLinkedData || {};
+  const places: (string | undefined)[] = [];
 
-  const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+  const getAuthorTagging = (data: BylineLinkedData | null | undefined) => {
+    const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
+      data || {};
 
-  const locationCreated = { '@place': location };
+    const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+    if (location !== undefined && places.indexOf(location)) {
```

Comment:
Is this a better way to check if the location is already in the list?
```suggestion
    if (!places.includes(location)) {
```